src/views/protected/MyAccount/MyAccountPage.tsx

```diff
@@ -0,0 +1,128 @@
+        minWidth: 300,
+    },
+    formContainer: {
+        marginTop: '2.3em',
+    },
+    comparisonText: {
+        paddingLeft: '1em',
+        lineHeight: '0.8em',
+        fontFamily: 'Arimo',
+    },
+    container: {
+        marginLeft: '1em',
+    },
+    select: {
+        width: '15em',
+    },
+}))
+
+const StyledFormControl = withStyles(() => ({
+    root: {
+        marginBottom: '1.3rem',
+    },
+}))(FormControl)
+
+const save = (name: string,
+              email: string,
+              password: string,
+              confirmPassword: string, firebase: Firebase) => {
+    if (password !== confirmPassword) {
+        alert('Passwords do not match')
+        return
+    }
+
+}
+/**
+ * @return {ReactNode}
+ */
+const MyAccountPage = ({ user = { firstName: '', lastName: '', email: '' } }): React.ReactNode => {
+    const classes = useStyles()
+    const [name, setName] = useState(`${user.firstName} ${user.lastName}`)
+    const [email, setEmail] = useState(user.email)
+    const [password, setPassword] = useState('')
+    const [confirmPassword, setConfirmPassword] = useState('')
+    return <div className={classes.root}>
+        <FirebaseContext.Consumer>
+            {(firebase: Firebase): React.ReactNode => <AppBar firebase={firebase} />}
+        </FirebaseContext.Consumer>
+        <div className={classes.formContainer}>
+            <Grid container
+                  direction="column"
+                  justify="flex-start"
+                  alignItems="center"
+                  style={{ width: '100vw', height: '100%', paddingTop: '1em' }}>
+                <Grid item xs={9}>
+                    <Typography style={{ fontSize: '2.5em' }}>
+                        My Account
+                    </Typography>
+                </Grid>
+                <Grid item xs={8} spacing={8} className={classes.container}>
+                    <StyledFormControl className={classes.formControl}>
+                        <InputLabel id="demo-mutiple-name-label">Name</InputLabel>
+                        <Input onChange={(event: React.ChangeEvent<HTMLInputElement>) => setName(event.target.value)}
+                               value={name} />
+                    </StyledFormControl>
+                </Grid>
+                <Grid item xs={8} spacing={8} className={classes.container}>
+                    <StyledFormControl className={classes.formControl}>
+                        <InputLabel id="demo-mutiple-name-label">Email</InputLabel>
+                        <Input onChange={(event: React.ChangeEvent<HTMLInputElement>) => setEmail(event.target.value)}
+                               value={email} />
+                    </StyledFormControl>
+                </Grid>
+                <Grid item xs={8} spacing={2} className={classes.container}>
+                    <StyledFormControl className={classes.formControl}>
+                        <FormHelperText>Enter a new password twice to change your password</FormHelperText>
```

Comment:
@thompchr forgot to add there's also an alignment issue with the Password label and the helper text (they are overlapping)